docs/apache-airflow-providers-google/operators/cloud/translate.rst

```diff
@@ -40,7 +40,7 @@ Using the operator
 
 Basic usage of the operator:
 
-.. exampleinclude:: /../../airflow/providers/google/cloud/example_dags/example_translate.py
+.. exampleinclude:: /../../tests/system/providers/google/translate/example_translate.py
```

Comment:
The destination location has changed from tests/system/providers/google/SERVICE/example_SERVICE.py to tests/system/providers/google/cloud/SERVICE/example_SERVICE.py